.samples/weather-to-logs-mi-manual-task copy.yaml

```diff
@@ -0,0 +1,12 @@
+displayName: Weather Report to Logs - Scheduled Task
+description: Log tabulated weather report with OpenWeatherMap data
+componentType: scheduled-task
+buildPack: wso2-mi
+repositoryUrl: https://github.com/wso2/choreo-samples/
+componentPath: /weather-to-logs-mi-scheduled-task
+thumbnailPath: /icons/weather-to-email-integration.svg
```

Comment:
Should we wait for a proper icon or can we go ahed with this?

Reply:
we can use this for now 
<img width="262" alt="image" src="https://github.com/wso2/choreo-samples/assets/59133724/ab8fcd30-999b-4d60-8789-2b82006fc310">

WDYT @chameerar ?

Reply:
Should be fine for now.